src/containers/monitor.jsx

```diff
@@ -171,7 +171,8 @@ class Monitor extends React.Component {
                 columnNumber = parseInt(prompt(msg), 10); // eslint-disable-line no-alert
             }
             const newListValue = rows.map(row => row[columnNumber - 1])
-                .filter(item => typeof item === 'string'); // CSV importer can leave undefineds
+                .filter(item => typeof item === 'string') // CSV importer can leave undefineds
+                .splice(0, LIST_ITEM_LIMIT);
```

Comment:
`splice` modifies the existing array, I believe. Is this okay/intentional?

Reply:
`filter` should deepcopy the array anyway, so it's okay.